### PR TITLE
Update LICENSE.txt

### PR DESCRIPTION
**Pull request by aadabi**

### CODE REVIEW

**Comment by aadabi** [comment]
Just added the University of California copyright.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
-Copyright (c) 2014-2016 Ali Adabi and Google Inc.
-
+"""
+Copyright (c) 2014-2016 Ali Adabi, Google Inc, and The Regents of the University of California
```
Comment (iangudger) [inline LICENSE.txt]: Should you still be listed as a copyright holder?

**Comment by aadabi** [comment]
Ian, Let's give this campus and see what they think.

**Comment by aadabi** [comment]
Couple of tests  failed in continuous-integration/travis-ci/push what's the solution to solve this?

File: LICENSE.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
-Copyright (c) 2014-2016 Ali Adabi and Google Inc.
-
-
+"""
```
Comment (iangudger) [inline LICENSE.txt]: Do we need these quotes?

**Comment by iangudger** [comment]
The Travis tests are currently failing due to this bug in Go:
https://github.com/golang/go/issues/16985

**Comment by iangudger** [comment]
The Go bug has been fixed and all tests now pass.

File: LICENSE.txt (hunk 2 above)
Comment (aadabi) [inline LICENSE.txt]: probably not but I have seen people do that.

**Comment by aadabi** [comment]
Thanks Ian, I am merging the Pull request, as I want to send it to the university, we can do minor changes later.

**Pull request merged**